### ISSUE TITLE
Arc 750 doesn't wake up from sleep mode

### ISSUE THREAD

**Issue by antonsherb** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#637]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Windows 11 (Entire OS/Any App)

### Processor / Processor Number [Required]

Intel Core i7 10700F

### Graphic Card [Required]

Arc A750

### GPU Driver Version [Required]

31.0.101.5074

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build Number [Required]

- [X] Windows 11 23H2
- [ ] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[igcit_ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/13665520/igcit_ssu.txt)

### Description and steps to reproduce [Required]

Video card is not able to wake up from sleep or hibernation. Fans are running and the boot/login sound is heard, but no image is produced. Only resetting the system fixes it.

Have tried multiple monitors, DP, and HDMI. All have the same issue. PCI Express Link State Power Management is turned off now. Have tried it on and off. BIOS is updated to most current version. 

### Device / Platform

Windows 11

**Comment by Vivek-Intel** [comment]
@antonsherb this is odd as I tried sleep-resume on couple of A750s I have and could not see any such issue so looks like system specific, can you help me with some more info 
1: what is VBIOS version on your card ? and install latest driver 101.5081
2: is it possible to use same GPU in other system and check if you can reproduce the issue
3: Are there any event viewer entries after sleep resume ? what I understand from your description is system does wake up, only display connected on GPU is not turning on ? system is not in hard hang state, please confirm.
4: you said it reproduce with different monitors, can you share all monitor you try so that we can check if we have any of the same monitor to simulate
5 : Can you try removing one ram stick and check if issue occurs

**Comment by SZim92** [comment]
Have you tried unplugging and replugging the DP/HDMI cable when it fails? I have encountered an issue where the display intermittently does not resume after sleep until the cable is unplugged and replugged. Might be related, but I haven't really looked into it yet.

**Comment by Vivek-Intel** [comment]
> i7 10700F

Thanks @antonsherb . I see you have already tried with disabling Native ASPM but can you give it a try with below specific setting and see if it resume properly from sleep.

BIOS -->Advance Mode-->Platform Misc configuration
PCI Express native Power management -- Enable
**Native ASMP -- Disable** 

_Under PCH PCI express_
ASPM -- L0sL1
L1 substate- Disabled

Save and reboot and then put system in sleep and wake up with keyboard and check if it resumes properly.

**Comment by antonsherb** [comment]
> > i7 10700F
> 
> Thanks @antonsherb . I see you have already tried with disabling Native ASPM but can you give it a try with below specific setting and see if it resume properly from sleep.
> 
> BIOS -->Advance Mode-->Platform Misc configuration PCI Express native Power management -- Enable **Native ASMP -- Disable**
> 
> _Under PCH PCI express_ ASPM -- L0sL1 L1 substate- Disabled
> 
[Windows System Log2.zip](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/13844042/Windows.System.Log2.zip)

> Save and reboot and then put system in sleep and wake up with keyboard and check if it resumes properly.

Thank you for the reply! Unfortunately this didn't work either. When the computer returned from sleep, the monitor flickered showing the input (Displayport) and then turned back off. Plugging and unplugging didn't help either.  Attached is the system log from this attempt.

**Comment by Vivek-Intel** [comment]
@antonsherb thanks for providing the logs. Can you uninstall googledrive and Intel Ethernet driver from system and check if you still run into the issue.
This is issue is very specific to your system as we were not able to replicate in any of the system available with us.

**Comment by antonsherb** [comment]
Thank you for the continued troubleshooting! Unfortunately uninstalling both also did not help. I tried to test it in safe mode to see if that made any difference but my system doesn't show sleep mode as an option in safe mode. Any other ideas short of a full system wipe?

**Comment by Vivek-Intel** [comment]
yes. sleep mode option wont be available in safe mode.
Can you try to put system in sleep mode without driver installed before we go for cleaning system so we can at-least isolate it to intel graphics driver

**Comment by Vivek-Intel** [comment]
@antonsherb did you get a chance to check suggested steps, and also reinstall the os ?

**Comment by antonsherb** [comment]
> @antonsherb did you get a chance to check suggested steps, and also reinstall the os ?

Yes, just tried it, unfortunately it still doesn't work. There's also no sleep mode if the driver is uninstalled, there is only hibernate. But on hibernate, resume also fails to wake up GPU properly, although the logon sound is heard through the speakers. 

I appreciate your support, but honestly, I'm just going to buy another graphics card. I don't want to spend any more time trying to sort this out.

**Comment by Vivek-Intel** [comment]
@antonsherb this is unfortunate but as you said its failing with hibernate even without graphics driver installed so it could be either OS or hardware problem.

you can check for warranty claims either from place of purchase or contacting Intel customer support from official website.

@IGCIT please help close this thread.
